docs/content/development/build.md

```diff
@@ -13,16 +13,18 @@ Building Druid requires the following:
 
 To do so, run these commands:
 
-```
+```bash
 git clone git@github.com:druid-io/druid.git
 cd druid
-mvn clean package
+mvn clean package -Pdist
 ```
 
 This will compile the project and create the Druid binary distribution tar under
 `distribution/target/druid-VERSION-bin.tar.gz`.
 
-This will also create a tarball that contains `mysql-metadata-storage` extension under 
+The difference between `mvn clean package` and `mvn clean package -Pdist` is that 
```

Comment:
How about this? https://gist.github.com/jihoonson/bffa89fd2a5f9473bb6467d3c52f12f6

Reply:
Hi, @jihoonson . Sorry for the delayed reply, this has been patched, PTAL.